test/AutoRest.TestServer.Tests/lro.cs

```diff
@@ -736,27 +718,27 @@ public class LroTest: TestServerTestBase
         });
 
         [Test]
-        [Ignore("Handle multiple responses: https://github.com/Azure/autorest.csharp/issues/413")]
         public Task LROPostAsyncNoRetrySucceeded() => Test(async (host, pipeline) =>
         {
             var value = new Product();
             var operation = await new LROsClient(ClientDiagnostics, pipeline, host).StartPostAsyncNoRetrySucceededAsync(value);
             var result = await operation.WaitForCompletionAsync().ConfigureAwait(false);
             Assert.AreEqual("100", result.Value.Id);
             Assert.AreEqual("foo", result.Value.Name);
-            Assert.AreEqual("Succeeded", result.Value.ProvisioningState);
+            // Final GET does not return provisioning state
+            Assert.AreEqual(null, result.Value.ProvisioningState);
```

Comment:
We should fix the test: https://github.com/Azure/autorest.testserver/blob/master/legacy/routes/lros.js#L782

I think it needs `res.end(outStr)` after the line I've linked.

Reply:
Technically, it isn't invalid to return no ProvisioningState after the final GET. But yes, I saw that line in routes, which is a copy-pasted line from other similar routes. Yeah, it should be fixed.

Reply:
Let's fix it before merging this PR.

Reply:
Created: https://github.com/Azure/autorest.testserver/pull/154

Reply:
Fixed!